.eslintignore

```diff
@@ -1,2 +1,4 @@
+build
+lib
 !/.storybook
 lib
```

Comment:
did you mean to duplicate lib?

Reply:
nope, somebody beat must have me to it.  I'll take it out